misc/theme-flexcolor/src/opnsense/www/themes/flexcolor/build/color_schemes/light and dark/default_scheme.css

```diff
@@ -0,0 +1,400 @@
+:root
```

Comment:
spaces in the directory names may pose an issue now or in the future: "light and dark"